workspaces/applications.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Applications"
+description:
+  Learn how to access web applications running on your remote workspace.
+state: beta
+---
+
+You can connect to web applications installed on your workspace using the
+applications specification file located at `/coder/apps/config.yaml` of the
+workspace filesystem.
+
+![Application Launcher](../assets/workspaces/applications.png)
+
+## Application specification file
+
+The application specification file allows you to define what Coder needs to
+launch the application. Within the file, you can specify the following for each
+application:
+
+```yaml
+# /coder/apps/config.yaml
+
+apps:
```

Comment:
are any of these optional, or are they all required?

Reply:
I believe these are all required. The only exception is one of `http` or `exec` needs to be specified.

cc @sreya to confirm.

Reply:
Looking through the code, it seems @scsmithr is right, everything top level field is required and 1 health check (can optionally do both).